### DISCUSSION TITLE
Unable to install spaCy version 2.2.3

### DISCUSSION THREAD

**Discussion by abs-xyz** [explosion/spaCy#10872] [Help: Installation] [answered]
I am unable to install spaCy version 2.2.3 using pip.
Earlier the error was along the lines "Microsoft Visual C++ 14.0 required", etc. Having installed that, the error is "The system cannot find the file specified".
I have attached the terminal output.
[error_output.txt](https://github.com/explosion/spaCy/files/8796090/error_output.txt)

**Comment by adrianeboyd** [comment, marked as answer]
In general, you'll have an easier time installing spacy v2.2.3 if you use a python version that was supported when it was released so that you can use precompiled wheels instead of compiling from source. It looks like python 3.8 or earlier is supported.

You can check for the python versions with wheels by looking at the "Download files" list on PyPI for this version: https://pypi.org/project/spacy/2.2.3/#files

If you do really want to compile from source, you'll need LLVM to compile blis in windows ([`choco install llvm`](https://community.chocolatey.org/packages/llvm) is one option):

https://github.com/explosion/cython-blis/tree/v0.4.1#installation
